app/serializers/score_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class ScoreSerializer < ActiveModel::Serializer
-  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :created_at, :updated_at
+  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :position, :created_at, :updated_at
```

Comment:
aaah okok hahaha we were talking the same thing but in different ways xD I was thinking you were going crazy haha sure thing do it and I will merge it